src/main/resources/schema/ans/0.5.3/author_operation.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.5.3/author_operation.json",
+  "description": "An operation on a author",
+  "type": "object",
+  "allOf": [{
+    "properties": {
+      "type": {
+        "description": "Identifies this as an ANS operation",
+        "type": "string",
+        "enum": [ "author-service" ]
+      },
+      "operation": {
+        "type": "string",
+        "description": "The identifier of the operation being performed",
+        "enum": [ "insert", "update", "delete" ]
+      },
+      "date": {
+        "description": "When the operation should be considered performed",
+        "type": "string",
+        "format": "date-time"
+      },
+      "id": {
+        "type": "string",
+        "description": "The id of the item being operated"
+      },
+      "organization_id": {
+        "type": "string",
+        "description": "The id of the organization"
+      },
+      "version": {
+        "type": "string",
+        "description": "The version of ANS this item is written in"
+      },
+      "body": {
```

Comment:
I don't think so.  That would require a change on the story-api and the content-api.